File: TakeFive/takeFive/src/main/java/com/teuskim/takefive/common/OmokLastData.java
package com.teuskim.takefive.common;

public class OmokLastData {

	private Pattern pattern;
	private int srow, scol, diffRow, diffCol;
	private boolean isWin;
	
	public void setPattern(Pattern pattern) {
		this.pattern = pattern;
	}
	
	public Pattern getPattern() {
		return pattern;
	}
	
	public void setResultPositions(int srow, int scol, int diffRow, int diffCol) {
		this.srow = srow;
		this.scol = scol;
		this.diffRow = diffRow;
		this.diffCol = diffCol;
	}
	
	public int getSrow() {
		return srow;
	}
	
	public int getScol() {
		return scol;
	}
	
	public int getDiffRow() {
		return diffRow;
	}

	public int getDiffCol() {
		return diffCol;
	}

	public boolean isWin() {
		return isWin;
	}

	public void setWin(boolean isWin) {
		this.isWin = isWin;
	}
	
}

File: TakeFive_kotlin/takeFive/src/main/java/com/teuskim/takefive/common/OmokLastData.kt
package com.teuskim.takefive.common

class OmokLastData {

    var pattern: Pattern? = null
    var srow: Int = 0
        private set
    var scol: Int = 0
        private set
    var diffRow: Int = 0
        private set
    var diffCol: Int = 0
        private set
    var isWin: Boolean = false

    fun setResultPositions(srow: Int, scol: Int, diffRow: Int, diffCol: Int) {
        this.srow = srow
        this.scol = scol
        this.diffRow = diffRow
        this.diffCol = diffCol
    }

}